kedro/framework/cli/project.py

```diff
@@ -110,8 +110,10 @@ def lint(
     click.secho(deprecation_message, fg="red")
 
     source_path = metadata.source_dir
+    project_path = metadata.project_path
     package_name = metadata.package_name
-    files = files or (str(source_path / "tests"), str(source_path / package_name))
+    # TODO: Detect whether tests are under project_path or source_path
```

Comment:
This "detection" would be needed if we want to make `kedro run` on Kedro 0.19 work with projects created on Kedro 0.18 and older